schema.sql

```diff
@@ -71,4 +71,16 @@ CREATE TABLE visits (
     vet_id INT REFERENCES vets(id),
     date_of_visit DATE,
     PRIMARY KEY (id)
-);
\ No newline at end of file
+);
+
+-- FIFTH PART --
+
+-- Add an email column to the owners table
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+-- Add indexes to the visits table animal_id
+CREATE INDEX idx_animal_id ON visits (animal_id);
+-- Add indexes to the visits table vet_id
+CREATE INDEX vet_id_asc ON visits(vet_id ASC);
+-- Add indexes to the owners table
+CREATE INDEX idx_email ON owners (email);
```

Comment:
[OPTIONAL]
- [ ] This is a nice-to-have suggestion as it won't improve the query that was reported as problematic in the requirements for this project.

  As email addresses are not case-sensitive, we might need to look for the emails in a case-insensitive way. There is an option to reflect that in the index by using:

```sql
CREATE INDEX ON owners ((lower(email)));
```